LGray is not a valid team and throws exception

**Describe the bug**
```
[14:54:22] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing 'bw autocreateteams' in com.andrei1058.bedwars.commands.bedwars.MainCommand(bw)
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:169) ~[patched_1.17.1.jar:git-Paper-411]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.dispatchCommand(CraftServer.java:869) ~[patched_1.17.1.jar:git-Paper-411]
	at org.bukkit.Bukkit.dispatchCommand(Bukkit.java:841) ~[patched_1.17.1.jar:git-Paper-411]
	at com.andrei1058.bedwars.commands.bedwars.subcmds.sensitive.setup.SetWaitingSpawn.execute(SetWaitingSpawn.java:56) ~[bedwars-plugin-22.3-SNAPSHOT.jar:?]
	at com.andrei1058.bedwars.commands.bedwars.MainCommand.execute(MainCommand.java:150) ~[bedwars-plugin-22.3-SNAPSHOT.jar:?]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[patched_1.17.1.jar:git-Paper-411]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.dispatchCommand(CraftServer.java:869) ~[patched_1.17.1.jar:git-Paper-411]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleCommand(ServerGamePacketListenerImpl.java:2262) ~[app:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:2073) ~[app:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:2054) ~[app:?]
	at net.minecraft.network.protocol.game.ServerboundChatPacket.handle(ServerboundChatPacket.java:46) ~[app:?]
	at net.minecraft.network.protocol.game.ServerboundChatPacket.a(ServerboundChatPacket.java:6) ~[app:?]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:56) ~[app:?]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[patched_1.17.1.jar:git-Paper-411]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:149) ~[app:?]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:23) ~[app:?]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1426) ~[patched_1.17.1.jar:git-Paper-411]
	at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:192) ~[patched_1.17.1.jar:git-Paper-411]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:122) ~[app:?]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1404) ~[patched_1.17.1.jar:git-Paper-411]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1397) ~[patched_1.17.1.jar:git-Paper-411]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:132) ~[app:?]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1375) ~[patched_1.17.1.jar:git-Paper-411]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1286) ~[patched_1.17.1.jar:git-Paper-411]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[patched_1.17.1.jar:git-Paper-411]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.IllegalArgumentException: No enum constant com.andrei1058.bedwars.api.arena.team.TeamColor.LGRAY
	at java.lang.Enum.valueOf(Enum.java:273) ~[?:?]
	at com.andrei1058.bedwars.api.arena.team.TeamColor.valueOf(TeamColor.java:29) ~[bedwars-plugin-22.3-SNAPSHOT.jar:?]
	at com.andrei1058.bedwars.api.arena.team.TeamColor.getChatColor(TeamColor.java:50) ~[bedwars-plugin-22.3-SNAPSHOT.jar:?]
	at com.andrei1058.bedwars.commands.bedwars.subcmds.sensitive.setup.AutoCreateTeams.execute(AutoCreateTeams.java:137) ~[bedwars-plugin-22.3-SNAPSHOT.jar:?]
	at com.andrei1058.bedwars.commands.bedwars.MainCommand.execute(MainCommand.java:150) ~[bedwars-plugin-22.3-SNAPSHOT.jar:?]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[patched_1.17.1.jar:git-Paper-411]
	... 25 more
```

**To Reproduce**
Steps to reproduce the behavior:
1. Set up a new arena in assisted mode
2. Scan the world for available teams

**Expected behavior**
No error to happen

**Screenshots**
Not relevant

**System (please complete the following information):**
Server Version: Paper - 1.17.1-411
 - Plugin Version 22.3.4
 Operating System: Linux - POP_OS

**Additional context**
I will make a PR to fix this today.

--- a/bedwars-api/src/main/java/com/andrei1058/bedwars/api/arena/team/TeamColor.java
+++ b/bedwars-api/src/main/java/com/andrei1058/bedwars/api/arena/team/TeamColor.java
@@ -234,7 +234,7 @@ public enum TeamColor {
                 name = "Red";
                 break;
             case "LIGHT_GRAY_WOOL":
-                name = "lGray";
+                name = "Gray";
                 break;
             case "BLUE_WOOL":
                 name = "Blue";
@@ -252,7 +252,7 @@ public enum TeamColor {
                 name = "Yellow";
                 break;
             case "GRAY_WOOL":
-                name = "Gray";
+                name = "Dark_Gray";
                 break;
         }
         return name;

--- a/bedwars-plugin/src/main/java/com/andrei1058/bedwars/commands/bedwars/subcmds/sensitive/setup/AutoCreateTeams.java
+++ b/bedwars-plugin/src/main/java/com/andrei1058/bedwars/commands/bedwars/subcmds/sensitive/setup/AutoCreateTeams.java
@@ -134,7 +134,7 @@ public class AutoCreateTeams extends SubCommand {
                     p.sendMessage("§6§lNEW TEAMS FOUND:");
                     for (String tf : found) {
                         String name = TeamColor.enName(tf);
-                        p.sendMessage("§f ▪ " + TeamColor.getChatColor(name) + name);
+                        p.sendMessage("§f ▪ " + TeamColor.getChatColor(name) + name.replace("_", " "));
                     }
                     p.spigot().sendMessage(Misc.msgHoverClick("§6 ▪ §7§lClick here to create found teams.", "§fClick to create found teams!", "/" + getParent().getName() + " " + getSubCommandName(), ClickEvent.Action.RUN_COMMAND));
                 }

